### ISSUE TITLE
List of locally downloaded maps show some in red: changed by scrolling

### ISSUE THREAD

**Issue by DFyson** [osmandapp/Osmand#3336]
In _Downloads_ → _Local_, scroll down and the text of some of the entries are in bright red. Scroll to bottom then back to top and other entries are then in red. Seems to be somewhat random and in all sections including voice prompts. I've noted this on OsmAnd+ 2.5.4 (2016-11-22) and OsmAnd~ 2.5.0#19490M (yesterdays nightly).

**Comment by vshcherb** [comment]
Red means broken or not indexed correctly. Please try to redownload, I can't find red local maps on my device.

**Comment by DFyson** [comment]
It appears in all categories (standard maps, online and caches maps, contour lines, wikipedia, and voice prompts). And once something is red, it will switch back too. For example a vector map will be highlighted in red, then scroll to the bottom of the list and back up and it is no longer red but another one will be. Scroll up and down enough and everything in the list becomes red. I have recently updated my maps, and I cleared the cache for the 'OsmAnd (online tiles)' and it still turned red.
